ascify art/Ascify-Art.py

```diff
@@ -91,13 +91,13 @@ def resource(relative_path):
 def operation():
     # main function that will do the magic
     global image, outputImage
-    
+
     if not file:
         return
-    
+
     def getChar(inputInt):
         return charArray[math.floor(inputInt*interval)]
-        
+

```

Comment:
Function `operation` refactored with the following changes:

- Remove unnecessary casts to int, str, float or bool